docs/zh/latest/plugins/opa.md

```diff
@@ -0,0 +1,318 @@
+---
+title: opa
```

Comment:
```suggestion
title: Open Policy Agent (OPA)
```

Reply:
I found that the name of the title is displayed in the sidebar, because the current sidebar display is the name of the plugin, so this should be displayed as opa more appropriate.

Reply:
Just have a look at OPA's website https://www.openpolicyagent.org/

I noticed they are using OPA as a short name, too. LGTM